src/Opdex.Platform.Application.Abstractions/Models/LiquidityPools/LiquidityPoolDto.cs

```diff
@@ -10,12 +9,9 @@ public class LiquidityPoolDto
         public string Name { get; set; }
         public Address Address { get; set; }
         public decimal TransactionFee { get; set; }
-        public bool StakingEnabled { get; set; }
-        public MiningPoolDto MiningPool { get; set; }
         public MarketTokenDto SrcToken { get; set; }
         public MarketTokenDto LpToken { get; set; }
-        public MarketTokenDto StakingToken { get; set; }
         public TokenDto CrsToken { get; set; }
```

Comment:
We have to retrieve these anyways for the assembler, at that point, we might as well return was my thinking.

Reply:
I think it'll be strange having a response which has the market address and liquidity pool address repeated many times. Could avoid this by using `TokenDto` rather than `MarketTokenDto`.

Reply:
hmmm it's a good point but then the question is _should_ we return a `TokenDto` when we're describing what are technically  `MarketToken`s?

Will give it some thought I can see it going both ways.